tests/e2e_tests/.eslintrc

```diff
@@ -0,0 +1,15 @@
+{
+  "parser": "@typescript-eslint/parser",
+  "extends": ["eslint:recommended", "plugin:@typescript-eslint/eslint-recommended", "plugin:@typescript-eslint/recommended"],
+  "rules": {
+    "prettier/prettier": 2,
+    "consistent-this": ["error", "self"],
+    "import/no-named-as-default": 0,
+    "no-console": 0,
+    "no-multiple-empty-lines": 2,
+    "no-prototype-builtins": 0,
+    "quotes": [2, "single", { "allowTemplateLiterals": true }],
+    "sonarjs/cognitive-complexity": 0,
+    "sonarjs/no-nested-template-literals": 0
```

Comment:
I would try to have it in, in case you want to go crazy with the backticks.

Reply:
we need this in e.g. the device connection snippet...

Reply:
I see. sorry for being annoying and challenging every line.